src/scenes/home/codeSchools/codeSchools.js

```diff
@@ -49,6 +49,10 @@ class CodeSchools extends Component {
             <br />
             We encourage you to check out the schools below, do your research, and ask fellow
             techies in our Slack Community.
+            <br />
+            <br />
+            <strong>Code Schools:</strong> if you&apos;d like your school to be listed here, please
+            fill out our <a href="http://op.co.de/code-school-request">request form</a>.
           </p>
```

Comment:
@pedrotp Looks good, thanks for the quick PR. I should have mentioned in the issue - we have an `OutboundLink` shared component located [here ](https://github.com/OperationCode/operationcode_frontend/blob/master/src/shared/components/outboundLink/outboundLink.js)we try to use for all external links.  Can you use this instead of the regular `<a href`>?